### PR TITLE
[DC-929] Add scripts copied from jade-data-repo-ui

### PR DESCRIPTION
**Pull request by s-rubenstein**
This adds the scripts that are copied from https://github.com/DataBiosphere/jade-data-repo-ui/tree/develop/tools/ui_integration, with the only change being to update the readme to use the new folder name, and changing the folder name to not reference the UI.

### CODE REVIEW

File: tools/datasetSnapshotHelperScripts/files/OMOPDataset/populate_omop_data.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+# (1) Set the gcp_with_access variable to a google project that you have job big query access on
+# (2) Set the gcp_project and gcp_dataset variables to the project and dataset that you want to pull data from
+# (3) Set the person_limit variable to the number of person records you want to pull.
+#       Note: Increasing this will very quickly increase the size of the tables, especially the concept table.
+
+# Running the script
+#1. If you need to set up a virtual python environment, you can run the following commands:
+#    * `python3 -m venv c:/path/to/myenv`
+#    * `source c:/path/to/myenv/bin/activate`
+#2. `cd jade-data-repo-ui/tools/ui_integration`
```
Comment (snf2ye) [inline tools/datasetSnapshotHelperScripts/files/OMOPDataset/populate_omop_data.py]: Can you update the path here?
```suggestion
#2. `cd jade-data-repo/tools/datasetSnapshotHelperScripts`
```

File: tools/datasetSnapshotHelperScripts/files/OMOPDataset/populate_omop_data.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+#3. `pip3 install -r requirements.txt`
+#4. `gcloud auth login <user>`
+#5. `cd files/OMOPDataset'
+#6  `python3 populate_omop_data.py`
+#7  `cd ../..` (go back to the ui_integration directory)
```
Comment (snf2ye) [inline tools/datasetSnapshotHelperScripts/files/OMOPDataset/populate_omop_data.py]:
```suggestion
#7  `cd ../..` (go back to the datasetSnapshotHelperScripts directory)
```

**Comment by s-rubenstein** [comment]
@snf2ye, I thought about changing the name of the script, but couldn't come up with a good name. Do you have ideas?

**Comment by snf2ye** [comment]
> @snf2ye, I thought about changing the name of the script, but couldn't come up with a good name. Do you have ideas?

Maybe setup_tdr_resources.py?

**Review by snf2ye — APPROVED** [review]

**Review by rjohanek — APPROVED** [review]

**Comment by s-rubenstein** [comment]
Tests can be ignored because there are no affected files.

**Pull request merged**